securitas/security/ipa.py

```diff
@@ -49,6 +64,21 @@ def group_add_member_manager(
             parse_group_management_error(data)
         return data['result']
 
+    def batch(self, methods=None, skip_errors=False):
+        """
+        Make multiple ipa calls via one remote procedure call.
+
+        :param methods: Nested Methods to execute.
+        :type methods: dict
+        :param skip_errors: Skip processing errors.
+        :type skip_errors: bool
+        """
+        data = self._request('batch', methods)
+        if not skip_errors:
+            for result in data['results']:
+                parse_batch_error(result)
```

Comment:
Oh, I had something different (and more complex) in mind:
- always go through the result list to detect errors
- if `raise_errors` is true, raise a BadRequest as soon as you find an error in the results
- if `raise_errors` is false, replace the error item with an instance of BadRequest in the list and return the list.

Is it clearer?